### PR TITLE
chore: upgrade Gradle to 5.6.1

### PR DESCRIPTION
**Pull request by clemstoquart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +87,7 @@ plugins {
     id "com.ekino.oss.gradle.plugin.quality" version "1.0.0"
 }
 ```
+
+### Requirements
+
+Gradle 5.3.1 and JDK 8 are required.
```
Comment (leomillon) [inline README.md:93]: Gradle `5.6`?

Comment (clemstoquart) [reply README.md:93]: Nope, it works well with Gradle 5.3.1 ;)

Comment (leomillon) [reply README.md:93]: ok

**Review by leomillon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Crow-EH) [reply README.md:93]: @clemstoquart Shouldn't we run the CI against this version to make sure it's still compatible in the future though ? There should be a way to have multiple builds against different versions.

Comment (clemstoquart) [reply README.md:93]: @Crow-EH This is exactly the purpose of `GradleVersionsCompatibilityTest`.

Comment (Crow-EH) [reply README.md:93]: Oh my bad. That's perfect :ok_hand:

Comment (Crow-EH) [reply README.md:93]: We should go all in and explicitely tell that those are the *minimum* versions, or even give the current range of tested versions.

Comment (clemstoquart) [reply README.md:93]: Maybe. Feel free to open an issue or a PR ;)

Comment (Crow-EH) [reply README.md:93]: Ok I'll do it in its dedicated PR.

**Review by Crow-EH — APPROVED** [review]

**Pull request merged**